src/plots/cartesian/constants.js

```diff
@@ -47,6 +47,12 @@ module.exports = {
     // delay before a redraw (relayout) after smooth panning and zooming
     REDRAWDELAY: 50,
 
+    // throttling limit (ms) for selectPoints calls
+    SELECTDELAY: 100,
+
+    // cache ID suffix for throttle
+    SELECTID: '-select',
```

Comment:
Non-blocking, but we should start thinking about moving selection things out of `plots/cartesian/` and into e.g. `component/fx/` or a new `component/select` module.